ddp-workspace/projects/ddp-dsm-ui/src/app/conditional-form-data/conditional-form-data.component.html

```diff
@@ -0,0 +1,8 @@
+<ng-container *ngIf="fieldSetting.displayType === 'TEXTAREA'">
+        <textarea maxlength="50000" placeholder="{{fieldSetting.columnDisplay}}"
```

Comment:
just wondering whether 50000 (50 thousand) is correct value here?

Reply:
That's going to change in another PR to come from backend as well

Reply:
But great catch! Thanks!